src/components/aeDivider/aeDivider.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <div :class="cssClass">
+  </div>
```

Comment:
I see that template for this component you have taken from `AeButton` or so. But I think better to write this component as simple as possible, so just `<div class="ae-divider" :class={ type } />` and remove `typeModifier` and `cssClass` computed variables.